queue_job/models/queue_job.py

```diff
@@ -226,7 +226,7 @@ def autovacuum(self):
         Called from a cron.
         """
         deadline = datetime.now() - timedelta(days=self._removal_interval)
-        jobs = self.search(
+        jobs = self.with_context(prefetch_fields=False).search(
```

Comment:
I was sure that the prefetch only happen the first time you access a field (other than id)?

Reply:
I will try to objective that with a test script (not in the module)